src/test/java/org/hydev/mcpm/client/database/SearchInteractorTest.java

```diff
@@ -39,39 +48,85 @@ private String formatStr(SearchPackagesResult result, String delim) {
     }
 
     @Test
-    void testSearchByNameSuccess() {
+    void testSearchByNameSuccessMatch() {
+        var result = database.search(
+                new SearchPackagesInput(SearchPackagesType.BY_NAME, "multiverse-core", true));
+
+        assert result.state() == SearchPackagesResult.State.SUCCESS;
+
+        var text = formatStr(result, ", ");
+        assert text.equals("Multiverse-Core");
+    }
+
+    @Test
+    void testSearchByNameSuccessNoMatch() {
         var result = database.search(
-                new SearchPackagesInput(SearchPackagesType.BY_NAME, "SkinsRestorer", true));
+                new SearchPackagesInput(SearchPackagesType.BY_NAME, "pp", true));
 
         assert result.state() == SearchPackagesResult.State.SUCCESS;
 
         var text = formatStr(result, ", ");
-        System.out.println(text);
-        assert text.equals("SkinsRestorer, SkinsRestorer");
+        assert text.equals("");
     }
 
     @Test
-    void testSearchByKeywordSuccess() {
+    void testSearchByKeywordSuccessMatch() {
         var result = database.search(
-                new SearchPackagesInput(SearchPackagesType.BY_KEYWORD, "offline online", true));
+                new SearchPackagesInput(SearchPackagesType.BY_KEYWORD, "players and", true));
 
         assert result.state() == SearchPackagesResult.State.SUCCESS;
 
         var text = formatStr(result, ", ");
-        System.out.println(text);
-        assert text.equals("CoordsOffline, StatusSigns, SkinsRestorer, PetShop, InventorySafe, SkinsRestorer");
+        assert text.equals("Holographic Displays, WorldGuard");
     }
 
     @Test
-    void testSearchByCommandSuccess() {
+    void testSearchByKeywordSuccessNoMatch() {
         var result = database.search(
-                new SearchPackagesInput(SearchPackagesType.BY_COMMAND, "al", true));
+                new SearchPackagesInput(SearchPackagesType.BY_KEYWORD, "pp", true));
 
         assert result.state() == SearchPackagesResult.State.SUCCESS;
 
         var text = formatStr(result, ", ");
+        assert text.equals("");
+    }
+
+    @Test
+    void testSearchByCommandSuccessMatch() {
+        var result = database.search(
+                new SearchPackagesInput(SearchPackagesType.BY_COMMAND, "/ungod", true));
+
+        assert result.state() == SearchPackagesResult.State.SUCCESS;
+
+        var text = formatStr(result, ", ");
+        assert text.equals("WorldGuard, Holographic Displays");
+    }
+
+    @Test
+    void testSearchByCommandSuccessNoMatch() {
+        var result = database.search(
+                new SearchPackagesInput(SearchPackagesType.BY_COMMAND, "pp", true));
+
+        assert result.state() == SearchPackagesResult.State.SUCCESS;
```

Comment:
`assertEquals(...)` please. There's a few of these regular asserts around this file.